### PR TITLE
Add Dockerfile and CI job for creating Docker image

### PR DESCRIPTION
**Pull request by yenugukeerthana**
This PR introduces a Dockerfile to the project, providing instructions for building a Docker image of the Avni Webapp. Additionally, the workflow configuration has been updated to facilitate the Dockerization process.

### CODE REVIEW

**Comment by ashusvnath** [comment]
@yenugukeerthana , the web-app is a set of static files served by avni-server. A separate container is not needed for the same.
If plan is to run web-app as a separate container, it(the contianer) needs to serve the web-app files via a web-server (eg: nginx) and also handle requests to web-app via proxying requests from web-app to avni-server.

**Review by ashusvnath — CHANGES_REQUESTED** [review]
please re-think the current containerisation approach

**Comment by mohan-13** [comment]
@ashusvnath This container does not serve the web-app static files by itself. Instead this acts as a placeholder which will mount the static file onto the avni-server container itself on the `/opt/openchs/static` path and eventually will be served only by avni-server container. 
Since this repo resides separately, we will use this docker image as a versioned artifact to be used.

**Comment by ashusvnath** [comment]
@mohan-13 : I haven't seen that done before, could you please show a sample docker invocation where one container accesses files from another running container.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+FROM alpine
+COPY build/ /build
+CMD ["sh", "-c", "cp -r /build/* /opt/openchs/static/ && sleep infinity"]
```
Comment (ashusvnath) [inline Dockerfile:3]: Will `/opt/openchs/static` exist in `alpine` . Is there an `mkdir` required in the `CMD` before the `cp` ?

Comment (mohan-13) [reply Dockerfile:3]: As you can see here, https://github.com/Bahmni-HWC/bahmni-india-package/commit/ee071e1ef13d7bfd50d1fe131dad0bad8b7baf0d, the volume mount will create the directory

Comment (ashusvnath) [reply Dockerfile:3]: @mohan-13 : This image needs to exist independently of volume mounts. IMO outside of that `docker-compose.yml`, running this image will fail without the `mkdir`

**Review by ashusvnath — APPROVED** [review]

**Pull request merged**